tools/deviceGen/output/features_union.h

```diff
@@ -0,0 +1,78 @@
+   // VK 1.0 Core
+    res.robustBufferAccess |= _rhs.robustBufferAccess;
+    res.geometryShader |= _rhs.geometryShader;
+    res.tessellationShader |= _rhs.tessellationShader;
+    res.depthBounds |= _rhs.depthBounds;
```

Comment:
spaces between entries of different groups (entry outside group counts as a group of a single entry) will make this more readable